src/style.css

```diff
@@ -1,3 +1,72 @@
+:root {
+  --main-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  height: 60px;
+}
+
+.nav {
+  display: flex;
+  height: 100%;
+  cursor: pointer;
+}
+
+.nav__ul {
+  list-style-type: none;
+  display: flex;
+  padding: 0;
   margin: 0;
 }
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  display: flex;
+  height: 100%;
+  margin-right: 20px;
+  align-items: center;
+}
+
+.nav__link--last {
```

Comment:
There is just one difference between styles applied to `.nav__link` and `.nav__link--last` which is margin-right set to 0 in --last.
If I were you, I would simply put only `margin-right: 0;` to --last
It would look like this:
`
.nav__link--last {
  margin-right: 0;
}
`
and in the HTML set two classes to the last element
`<a class="nav__link nav__link--last" href="#video">VIDEO</a>`

Reply:
This approach is better because if you wanted to change some styles of that link, you would only change them in one place - `.nav__link`